setup.py

```diff
@@ -55,8 +55,8 @@
         arq=arq.cli:cli
     """,
     install_requires=[
-        'redis>=4.2.0rc2',
-        'click>=6.7',
+        'redis>=4.2.0',
+        'click>=6.7,<8.1.0',
```

Comment:
is this required?

Reply:
The `click>=6.7,<8.1.0`, no, not anymore. There was an issue with black (https://github.com/psf/black/issues/2964). But it looks it's ok now. I removed it. The `redis>=4.2.0` is needed if we include redislite on tests.